components/organisms/Header/Header.js

```diff
@@ -16,22 +18,37 @@ const HeaderWrapper = styled.header`
 `;
 
 const Profile = styled.img`
-  height: 100%;
+  height: 40px;
 `;
 
-const Header = () => {
-  const profile = false;
+const Links = styled.div`
+  display: flex;
+  align-items: center;
+`;
 
+const Header = ({ user, signOut }) => {
   return (
     <HeaderWrapper>
       <Logo />
-      {profile ? (
-        <Profile src={`${process.env.ASSET_HOST}/images/avatar-placeholder.png`} alt="avatar" />
-      ) : (
-        <Link passHref route="login">
-          <a>Log In</a>
-        </Link>
-      )}
+      <Links>
+        {!user && (
+          <Link route={LOGIN.pattern}>
+            <a>Log In</a>
+          </Link>
+        )}
+        {!!user && (
```

Comment:
maybe better check user, not boolean value of user?

Reply:
This variable can have two possible values: `object` or` undefined`. I think this is the shortest way